static/js/components/email/EmailCompositionDialog.js

```diff
@@ -165,8 +165,8 @@ export default class EmailCompositionDialog extends React.Component {
 
   onEditorStateChange = (editorState: EditorState) => {
     const { updateEmailBody } = this.props;
-    this.setState({ editorState });
     updateEmailBody(editorState);
+    this.setState({ editorState });
```

Comment:
good catch! I'm guessing we were getting a race condition with `componentWillReceiveProps`, where the `updateEmailBody` call was causing a prop change and then we were setting the state twice. it's nice when there's a simple solution to things :smile: